### DISCUSSION TITLE
Are there any examples of the Vercel API in action?

### DISCUSSION THREAD

**Discussion by esteban89** [vercel/vercel#4610] [Help] [answered]
I'm trying to upload and deploy directly from Laravel into Vercel. Are there any examples I can see on how to do it? I'm reading the docs but there are some things that aren't quite clear to me.

Thanks,
Esteban

**Comment by styfle** [comment, marked as answer]
You can find our official examples in this repo: https://github.com/vercel/vercel/blob/master/examples

For example, heres the "Vercel API in action" using a serverless function: https://github.com/vercel/vercel/blob/master/examples/svelte/api/date.js

If you're looking for PHP examples, try this repo: https://github.com/juicyfx/now-examples
